ops/environments/000_cross_env_vars

```diff
@@ -0,0 +1,139 @@
+#ansistrano_git_identity_key_path: "keys/deploy" # If specified this file is copied over and used as the identity key for the git commands, path is relative to the playbook in which it is used
+ansistrano_before_symlink_tasks_file: "{{ playbook_dir }}/roles/deploy/tasks/before_symlink.yml"
+ansistrano_after_symlink_tasks_file: "{{ playbook_dir }}/roles/deploy/tasks/after_symlink.yml"
+
+capistrano_user: deploy
+capistrano_group: deploy
+deploy_target: development
+
+# Database
+db: postgres
+db_adapter: postgresql
+hydra_db: hydra
+hydra_db_user: horton
+hydra_db_pass: penguins;13f1
+hydra_db_host: postgres 
+hydra_db_port: 5432
+
+# Deploy
+fedora_base_path: /{{ deploy_target }}
+project_name: horton
+shared_root: /opt/{{ project_name }}/shared
+
+# Keepalived
+shared_internal_ip: 172.21.62.59
+shared_external_ip: 169.228.62.100
+
+keepalived_sync_groups:
+  haproxy:
+    instances:
+      - internal
+keepalived_scripts:
+  haproxy_check_script:
+    check_script: "killall -0 haproxy"
+  pingable_check_script:
+    check_script: "ping -c 1 -p {{ keepalived_check_port }} {{ keepalived_protected_address }}  1>&2"
+    # Interval is the time in seconds between each check
+    interval: 120
+    # Fall is the time to mark the keepalived status as not ok
+    fall: 2
+    # Rise is the time to mark the keepalived status as back to ok
+    rise: 4
+    # Allow the check script to complete within a timeout
+    timeout: 20
+
+keepalived_instances:
+  internal:
+    interface: "{{ vrrp_nic }}"
+    state: "{{ (play_hosts.index(inventory_hostname) == 0) | ternary('MASTER','BACKUP') }}"
+    virtual_router_id: 42
+    priority: "{{ (play_hosts.index(inventory_hostname) == 0) | ternary('100','50') }}"
+    track_scripts:
+      - haproxy_check_script
+      - pingable_check_script
+    vips:
+      - "{{ vip }}/24 dev {{ vrrp_nic }}"
+
+# Nodejs
+nodejs_install_npm_user: "{{ capistrano_user }}"
+
+# Redis
+redis_address: "{{ hostvars[groups['horton_sidekiq_master'][0]]['ansible_eth0']['ipv4']['address'] }}"
+
+# Postfix
+postfix_enabled: yes
+postfix_myhostname: "{{ inventory_hostname }}"
+postfix_dkim_domain: "{{ inventory_hostname }}"
+postfix_smtp_use_tls: yes
+postfix_smtp_tls_cafile: /etc/ssl/certs/ca-certificates.crt
+postfix_smtp_tls_loglevel: 1
+postfix_smtp_tls_security_level: encrypt
+postfix_smtp_tls_note_starttls_offer: yes
+
+# Ruby
+ruby_ver: '2.4.1'
+rvm1_rubies:
+  - 'ruby-2.4.1'
+rvm1_install_flags: '--auto-dotfiles'     # Remove --user-install from defaults
+rvm1_install_path: /usr/local/rvm         # Set to system location
+rvm1_user: root
+
+# Solr
+solr_version: 6.6.0
+# solr_url: horton_solr_1
+solr_url: "{{ shared_internal_ip }}"
+solr_port: 8983 
+solr_context: solr
+solr_core: collection1
+zookeeper_hosts: "{{groups['horton_solr']}}"
+
+# SSH Keys
+ssh_keys:
+  - winescout
+  - smcfarlane
+  - orangewolf
+# TODO UCSD users
```

Comment:
Yeah we should talk about this. Once this is up and running cleanly, I wonder how much SSH access will/should be necessary

Reply:
just the people who can deploy need access to the deploy user via ssh (which is what this is about)